docs/cloud/azureai/tracing/index.md

```diff
@@ -0,0 +1,84 @@
+# Tracing from local to cloud
+
+:::{admonition} Experimental feature
+This is an experimental feature, and may change at any time. Learn [more](../../../how-to-guides/faq.md#stable-vs-experimental).
+:::
+
+Prompt flow [tracing feature](../../../how-to-guides/tracing/index.md) enables users to trace LLM calls, functions and even LLM frameworks. Besides, with Azure extension installed, prompt flow can also log traces to an Azure ML workspace or Azure AI project, which makes it possible to share traces with your team members.
+
+## Installing the package
+
+```console
+pip install "promptflow[azure]>=1.11.0"
+```
+
+## Set cloud destination
+
+To log traces to cloud, first of all, you need an [Azure ML workspace](https://learn.microsoft.com/en-us/azure/machine-learning/concept-workspace?view=azureml-api-2) or [Azure AI project](https://learn.microsoft.com/en-us/azure/ai-studio/how-to/create-projects). Then, you can set the destination via below CLI command:
```

Comment:
I notice in chenlu's doc, she explicity added an az login before `pf config set`, what's the error if we set when logged out?

Reply:
![image](https://github.com/microsoft/promptflow/assets/38847871/244a8e4f-db0e-43ce-ae60-5b2a680fc758)

some trace backs, but finally an error message to hint user to execute `az login`

Reply:
however, maybe we can highlight this in our doc, will add this.

Reply:
and had better to mention what's the default trace destination, local?

Reply:
had better mention that we will have the sth setup on Azure for tracing feature enable when user first time set to cloud (like the cosmos, but don't indicate it is cosmos to avoid future change) 

and what errors users may encounter when cosmos setup failed? how to troubleshoot?

Reply:
added highlight for azure cli login

Reply:
describe some details in section Storage

currently, we returns error messages from service side, so no more cases can be shared here for now.